bin/fix-dbs-parentage

```diff
@@ -57,7 +57,7 @@ def getMissingParentSummaryUsingBlock(datasetList, fileFix=False, insertFlag=Fal
     start = int(time.time())
 
     if insertFlag:
-        answer = raw_input("fixing DBS parentage: Are you sure?(y/n): ").lower()
+        answer = input("fixing DBS parentage: Are you sure?(y/n): ").lower()
```

Comment:
Hmm, I know it is more compatible with Python 3, when I tried it before it didn't work (Aside it evaluates the input).  I will try again.

Reply:
Ahhh, I just noticed it indeed complains in ipython. But if you type "y" instead of y, it works, apparently. So one needs to enclose the input in quotes, which is weird! Is it acceptable, otherwise we'll keep getting this failure in jenkins.